src/pages/find-offers/FindOffers.jsx

```diff
@@ -34,6 +53,13 @@ const FindOffers = () => {
     setSearchParams({ view, role })
   }, [view, role, setSearchParams])
 
+  useEffect(() => {
+    fetchData({
+      limit: offersPerPage
+      // TODO: Use skip param to implement pagination
+    })
```

Comment:
Add `fetchData` as dependency or skip es lint rule fir the next line if adding is impossible.